R/dvhydrograph-data.R

```diff
@@ -163,7 +180,7 @@ extendStep <- function(toPlot){
   isStep <- 'type' %in% names(toPlot) && toPlot[['type']] == "s"
   
   if(isStep){
-    toPlot$x <- c(toPlot$x,  tail(toPlot$x, 1) + 90000) #this is 2400, if changing to POSTLT, need to use lubridate
+    toPlot$x <- c(toPlot$x,  tail(toPlot$x, 1) + 60*60*24) #this is 2400, if changing to POSTLT, need to use lubridate
```

Comment:
can you change the comment associated with this to make it a little clearer? like what does this even do? "this is 2400" doesn't really mean much - I realize that's what was there before, but would be nice to update!